cpp/include/cudf/detail/aggregation/aggregation.cuh

```diff
@@ -139,9 +140,13 @@ struct update_target_element<
   {
     if (source_has_nulls and source.is_null(source_index)) { return; }
 
-    using Target = target_type_t<Source, aggregation::MIN>;
-    atomicMin(&target.element<Target>(target_index),
-              static_cast<Target>(source.element<Source>(source_index)));
+    if constexpr (cudf::has_atomic_support<Source>()) {
+      using Target = target_type_t<Source, aggregation::MIN>;
+      atomicMin(&target.element<Target>(target_index),
+                static_cast<Target>(source.element<Source>(source_index)));
+    } else {
+      cudf_assert(false and "Source has no atomic support.");
+    }
```

Comment:
This isn't what I meant. The `enable_if` statement should be updated such that types that don't have atomic support never enter this code path.

Reply:
You can just replace the `is_fixed_width<Source>` statement with `has_atomic_support<Source>`.

Reply:
Fixed.